vulcanai/tests/models/conftest.py

```diff
@@ -317,3 +317,23 @@ def multi_input_cnn_data(conv2D_net, conv3D_net, multi_input_dnn_data):
         (TensorDataset(torch.rand(size=[10, *conv3D_net.in_dim])), True, False),
         multi_input_dnn_data
     ])
+
+
+@pytest.fixture(scope="module")
+def dnn_class_single_value_2():
```

Comment:
what does the 2 represent? please add a bit more information in the class documentation where this fixture is being used

Reply:
this may change on how you update the test to use the already embedded test csv file